docker/script/bootstrap.sh

```diff
@@ -29,6 +29,15 @@ adduser -s /bin/bash -d "${AIRFLOW_USER_HOME}" airflow
 # install watchtower for Cloudwatch logging
 pip3 install $PIP_OPTION watchtower==1.0.1
 
+pip3 install $PIP_OPTION apache-airflow-providers-tableau==1.0.0
```

Comment:
Do we want to constrain so tightly?

Reply:
I noticed there was a [breaking change](http://phoenix2.yizimg.com/apache/airflow/pull/15667) in the provider packages recently. This change probably moved the provider versioning from so-and-so-provider-1.0 to a so-and-so-provider-2.0.

Do a search on the key word Breaking Changes [here](https://airflow.apache.org/docs/apache-airflow-providers-tableau/stable/index.html). This tries to install the updated 2.1.x version on the image. Since the local runner is for a 2.0.2 environment, this could potentially change the underlying airflow version or install a provider version that is not compatible with this airflow version. Hence had to specify the compatible provider version.